mne/utils/numerics.py

```diff
@@ -1056,3 +1057,58 @@ def restore(self, val):
         else:
             loc = np.searchsorted(self.indices, idx)
             self.indices.insert(loc, idx)
+
+
+def prefix(number, ten_hundred=False):
```

Comment:
just skimming, but this could potentially useful for my stale PR to make a "unit" parameter available to the get_data methods --> https://github.com/mne-tools/mne-python/pull/8476

Reply:
```suggestion
def unit_prefix(number, ten_hundred=False):
```
prefix is not specify enough

Reply:
```suggestion
def unit_prefix(number, ten_hundred=False):
```
prefix is not specify enough